### ISSUE TITLE
Error: Argument 2 must be length 360, not 361

### ISSUE THREAD

**Issue by ngastoncyrille** [chrisschuerz/SWAT_D_2019#1]
Hi,
I have just discovered the SWATplusR package, and I am using it for the calibration of my SWAT model. However, I have some difficulty. Indeed, when exporting my project (TxtInOut) I have an error message:
proj_path <- "C:/TxtInOut"
q_out <- run_swat2012(project_path = proj_path,
        output = define_output(file ="rch",
        variable = "FLOW_OUT",unit = 25))
Building 1 thread in' C:/TxtInOut/.model_run': 
 Completed 1 thread in 2S                                                   
Performing 1 simulation on 1 cores: 
 Completed 1 simulation in 5S                                               
Error: Argument 2 must be length 360, not 361

**Comment by chrisschuerz** [comment]
Hi,

from your error message I only can make assumptions. But it seams to me that you simulated a leap year, but SWATplusR did not recognize it to be one. This could cause that discrepancy of one day. To really get to the core of the problem, I would need an example to reproduce the error. Either the error can be generated with the demo data set. Then I can try it myself, or you provide your model setup to follow what you did.

I would also ask you to post issues regarding SWATplusR in the issue section of the R package. This repository is an outdated repository that I used for an introductory course. To give other users the chance to look up your issue, if they encounter the same problem, please again post it in the SWATplusR repository and maybe provide a reproducible example and we follow up from there.

Cheers, Christoph

**Comment by ngastoncyrille** [comment]
Hi,
I received your message well. However, I had a solution to my problem. But
I had to rethink my simulation, whose outputs were in months and days. And
the error message has disappeared. With regard to your recommendation on
the distribution platform, I take note.
I use your package for the calibration of flows in the Nakanbé basin at the
wayen station (BURKINA FASO). If I succeed in doing so, I intend to put the
procedure in pdf format for the benefit of my classmates in training with
me. Indeed, I am a student in hydrology at the Agrhymet Regional Centre
(CRA), and I am working on this model as part of my final thesis.
THANK YOU very much for your availability.
Best regards NEBIE.
